tests/lib/eslint-tester.js

```diff
@@ -329,6 +329,27 @@ describe("ESLintTester", function() {
         });
     });
 
+    it("should pass-through the parser to the rule", function() {
```

Comment:
I did not wanted to add a new devDependency just for one test, but if you are ok this that can do it.

Also, how can I test inside the rule that we are using the right parser ? I think the parser is not exposed to the rule, is it ?